addons/web_editor/static/src/scss/secondary_variables.scss

```diff
@@ -48,7 +48,18 @@ $o-theme-color-palette: map-merge($o-base-theme-color-palette, o-map-omit($o-the
 // white-50 but does not allow overridding that with variables).
 $o-gray-color-palette: o-safe-nth($o-gray-color-palettes, $o-gray-color-palette-number, ()) !default;
 $o-gray-color-palette: map-merge($o-transparent-grays, o-map-omit($o-gray-color-palette));
-$o-gray-color-palette: map-merge($o-base-gray-color-palette, o-map-omit($o-gray-color-palette));
+$gray-hue: 210 !default;
+$gray-extra-saturation: 0 !default;
+@function adjust-gray-color($gray) {
+    @return adjust-color(change-color($gray, $hue: $gray-hue * 1deg), $saturation: $gray-extra-saturation * 1%);
+}
+$o-adjusted-gray-color-palette: ();
+@each $key, $color in $o-base-gray-color-palette {
+    $o-adjusted-gray-color-palette: map-merge($o-adjusted-gray-color-palette,
+            (base-#{$key}: $color, $key: adjust-gray-color($color))
+    );
+}
+$o-gray-color-palette: map-merge($o-adjusted-gray-color-palette, o-map-omit($o-gray-color-palette));
```

Comment:
The values in the grays palette are use to generate classes. If you had new keys "base-XXX" inside... there will be useless classes which are generated.

Also I think we will get rid of the multiple gray color palette at some point, but meanwhile there is not point in taking the grays from a palette and mixing them with base grays whose hue and saturation are tweaked. Compute the palette just like before:
```scss
$o-gray-color-palette: map-merge($o-base-gray-color-palette, o-map-omit($o-gray-color-palette));
```
And then adjust each gray inside that one.